projects/fourier/lib/fourier/commands/lint.rb

```diff
@@ -55,14 +54,21 @@ def lockfiles
       option :fix, desc: "When passed, it fixes the issues", type: :boolean, default: false
       def all
         Services::Lint::Lockfiles.call
-        Services::Lint::Tuist.call(fix: options[:fix])
+        lint_tuist(fix: options[:fix])
         Services::Lint::Tuistbench.call(fix: options[:fix])
         Services::Lint::Fixturegen.call(fix: options[:fix])
         Services::Lint::CocoapodsInteractor.call(fix: options[:fix])
         Services::Lint::Fourier.call(fix: options[:fix])
         Services::Lint::Cloud.call(fix: options[:fix])
         Services::Lint::Backbone.call(fix: options[:fix])
       end
+
+      no_commands {
+        def lint_tuist(fix: false)
+          Services::Lint::Tuist.call(fix: fix)
+          Services::Format::Swift.call(fix: fix)
+        end
+      }
```

Comment:
Fixes `./fourier lint all` to actually do all lints (we were missing swift format step for tuist repo)